### DISCUSSION TITLE
How to part the file which is not in default fomat

### DISCUSSION THREAD

**Discussion by prabhir** [jsdoc2md/jsdoc-to-markdown#245] [Ideas]
/**
 * A quite wonderful function. This is not the function which we are looking for
 * @param {object} cloak
 * @param {object} - Security
 */

(function protection (cloak, dagger) {

})

**Comment by 75lb** [comment]
First, please test your documentation in jsdoc - if it does not work in jsdoc it will not work in jsdoc2md.. Does you documentation work using jsdoc?
